workers/unity/Packages/com.improbable.gdk.core/Systems/WorkerDisconnectCallbackSystem.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Enables users to add a callback onto the disconnection event.
```

Comment:
This is used exclusively by the Worker class for it's event. It's what backs that. Although this can be used by users to add a callback.